projects/ngx-interactive-paycard-lib/src/lib/card/card.component.ts

```diff
@@ -3,6 +3,8 @@ import { trigger, transition, state, animate, style } from '@angular/animations'
 
 import { CardModel } from '../shared/card-model';
 import { FocusedElement } from '../shared/focused-element';
+import {CardLabel} from '../shared';
+
```

Comment:
Could you please add a whitespace before and after `CardLabel` to follow the conventions. And please remove the unnecessary line.